vizro-core/src/vizro/static/css/tabs.css

```diff
@@ -10,6 +10,7 @@
   align-self: stretch;
   border-bottom: 1px solid var(--border-subtle-alpha-01);
   display: flex;
+  flex-wrap: nowrap;
```

Comment:
The tabs were previously wrapping on a new line if there was not enough space, which made them unusable as they were overlayed on other components. Better to have it scrollable rather than wrapping on a new line given our grid-layout